scripts/test-ssr/src/utils/getExportFromFile.test.ts

```diff
@@ -1,33 +1,30 @@
 import * as fs from 'fs';
 import * as path from 'path';
+
 import * as tmp from 'tmp';
 
-import { getImportsFromIndexFile } from './getImportsFromIndexFile';
+import { getExportFromFile } from './getExportFromFile';
 
-describe('getImportsFromIndexFile', () => {
+describe('getExportFromFile', () => {
   it('parses export and returns information to create imports', async () => {
-    const template = `
-export { Foo } from './foo/foo';
-export { Bar } from './bar/bar';
-    `;
+    const template = 'export const Bar = () => <div>Bar</div>';
```

Comment:
- Before: we will `index.stories.tsx` to find stories
- Now: we have files with stories from a glob, we will parse every file to find its export